controllers/api.controllers.js

```diff
@@ -0,0 +1,5 @@
+const endpoints = require("../endpoints.json")
+
+exports.getEndpoints = (req, res, next) => {
+    res.status(200).send(endpoints)
```

Comment:
data sent back from controllers should be sent back on a meaningful key ie. `.send({endpoints})`